adafruit_sdcard.py

```diff
@@ -405,18 +405,18 @@ def readblocks(self, start_block, buf):
             # CMD17: set read address for single block
             # We use _block_cmd to read our data so that the chip select line
             # isn't toggled between the command, response and data.
-            if self._block_cmd(17, start_block, 0, response_buf=buf) != 0:
+            if self._block_cmd(17, start_block, 0x3b, response_buf=buf) != 0:
```

Comment:
This CRC will need to change.

Reply:
ahhhhh  Ok now I understand.  Sorry I was not thinking that start_block contained arguments. I was just thinking that start_block WAS the argument and therefore not changing.  :)